### ISSUE TITLE
CONFIG_OVERLAY_FS missing from arm64 kernel 5.4ff

### ISSUE THREAD

**Issue by buczek** [dentproject/dentOS#64]
```
debian@debian:~/OpenNetworkLinux$ grep "OVERLAY_FS=" packages/base/any/kernels/*/configs/arm64*/*.config
packages/base/any/kernels/4.9-lts/configs/arm64-all/arm64-all.config:CONFIG_OVERLAY_FS=y
debian@debian:~/OpenNetworkLinux$ grep "OVERLAY_FS is not set" packages/base/any/kernels/*/configs/arm64*/*.config
packages/base/any/kernels/5.10-lts/configs/arm64-all/arm64-all.config:# CONFIG_OVERLAY_FS is not set
packages/base/any/kernels/5.6-lts/configs/arm64-all/arm64-all.config:# CONFIG_OVERLAY_FS is not set
debian@debian:~/OpenNetworkLinux$ 
```

As a result, only the INSTALLED_INSTALLER is working.

**Comment by sonoble** [comment]
As an end user, are you looking to use the SWI based dentOS?

**Comment by buczek** [comment]
Sorry, I'm not sure I understand "SWI based dentOS" in this context. I'm evaluating DentOS as a basis for our future switch infrastructure. Yes, of course, I want to build software images and install them via ONIE on our switches. So we already did with our test switches. I'm not aware of alternatives.

Regarding this issue: Currently, the doc directory in this repository is still just the one from ONL, which includes PersistWorkflow.md [1]. So either DentOS wants to drop that feature, in which case PersistWorkflow.md and maybe the rest of the doc directory is a red herring which should be updated or removed, or DentOS wants to keep it, in which case the OVERLAY_FS should no be disabled.

So this issue is an about an assumed contradiction of the docs and  the code in this repository.

[1]: https://github.com/dentproject/dentOS/blob/c776a7138dc458befe4e559b1b9224ba699bcdf9/docs/PersistWorkflow.md

**Comment by sonoble** [comment]
When you run in swi mode, you load the rootfs each time you boot. In this mode you might use a controller or ztp to push a configuration to the system. In installation mode, the system is persistent.

We can put overlayfs in, it's needed for docker.

**Comment by buczek** [comment]
Yes, this is definitely a vision I have: All switches start from the same read-only file system every time. All individual configuration is provided from a central repository. The configuration should, of course, be cached on the switch to avoid delay and (more important) network dependencies during reboots. OverlayFS might be useful to guarantee that I don't have some unwanted and untracked  "customization"  on the writable rootfs of individual switches, which bites me when I replace broken units.
